docs/rule/no-redundant-landmark-role.md

```diff
@@ -0,0 +1,35 @@
+# no-redundant-landmark-role
+
+If a landmark element is used, any role provided will either be redundant or incorrect.
+Add support for landmark elements to ensure that no role attribute is placed on any of
+the landmark elements, with a few exceptions.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<header role="banner"></header>
+```
+
+```hbs
+<main role="main"></main>
+```
+
+```hbs
+<aside role="complementary"></aside>
+```
+
+```hbs
+<nav role="navigation"></nav>
+```
+
+This rule **allows** the following:
+
+```hbs
+<form role="search"></form>
+```
+
+```hbs
+<footer role="contentinfo"></footer>
+```
```

Comment:
Please add the reference material here. 

- Landmark Roles (WAI-ARIA spec): https://www.w3.org/WAI/PF/aria/roles#landmark_roles
- Using ARIA landmarks to identify regions of a page- https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11

Reply:
done